src/test/TestDASHTree.cpp

```diff
@@ -187,30 +187,30 @@ TEST_F(DASHTreeTest, CalculateCorrectSegmentNumbersFromSegmentTimeline)
 
 TEST_F(DASHTreeTest, CalculateCorrectSegmentNumbersFromSegmentTemplateWithPTO)
 {
-  tree->mock_time = 1617223929L;
+  tree->mock_time = 1616751595L; //1616715843 (avail start) + 35652 (period start) + 100 (in future)
 
   OpenTestFile("mpd/segtpl_pto.mpd", "", "");
 
   adaptive::SPINCACHE<adaptive::AdaptiveTree::Segment> segments =
       tree->periods_[0]->adaptationSets_[0]->representations_[0]->segments_;
 
-  EXPECT_EQ(segments.size(), 451);
-  EXPECT_EQ(segments[0]->range_end_, 404305525);
-  EXPECT_EQ(segments[450]->range_end_, 404305975);
+  EXPECT_EQ(segments.size(), 25); //25 * 4 = 100 seconds
```

Comment:
I'm not understanding the 100 second result here for both manifests - TSBD for this is 1800 and 120 for the other, the old values for segment size seem correct.